Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml

```diff
@@ -0,0 +1,1612 @@
+category: Case Management
+commonfields:
+  id: OpsGenieV3
```

Comment:
why not have it same as the displayname `OpsGenie v3`?
just for same string all around.

Reply:
If I change the is I need to change also the name of the yml file. Is it necessary?

Reply:
no, just easier to have id==name==displayname